godtools/App/Features/Lessons/Lessons/LessonsView/LessonsViewModel.swift

```diff
@@ -81,8 +82,10 @@ extension LessonsViewModel {
             .receiveOnMain()
             .sink { [weak self] lessons in
                 
-                self?.isLoading = lessons.isEmpty
-                self?.lessons = lessons
+                withAnimation {
```

Comment:
Hey @rachaelblue do you know if the withAnimation block will work keeping it in the View?  It doesn't seem correct to have that in the ViewModel since animation logic is a responsibility of the View.

Reply:
I can try adding implicit animations to the view instead, I believe that should have the same effect.